src/config/dynamic-theme-fixes.config

```diff
@@ -13005,6 +13005,46 @@ CSS
     background-color: ${#cfcfcf} !important;
 }
 
+
+================================
+
+mail.eni.it/owa
+
+IGNORE IMAGE ANALYSIS
+.image-font-png
```

Comment:
Maybe it can be replaced with one of those 3 selectors?
`[class^="image-"]`
`[class^="-png"]`
`[class^="image-"][class^="-png"]`